reconcile/gitlab_labeler.py

```diff
@@ -30,6 +85,13 @@ def guess_labels(project_labels, changed_paths):
                        and t not in ignore_tokens]
             if matches:
                 guesses.add(label)
+    apps = get_app_list()
+    parent_apps = get_parents_list()
+    onboarding_status = guess_onboarding_status(changed_paths, apps,
+                                                parent_apps)
+    logging.info(onboarding_status)
```

Comment:
we are already logging in line 111, no need to log in internal functions. essentially we log only when the integration does something (add labels in this example).